app/src/main/java/net/gsantner/markor/activity/DocumentEditFragment.java

```diff
@@ -662,10 +654,13 @@ public boolean onBackPressed() {
         return false;
     }
 
+    public boolean saveDocument() {
+        return saveDocument(false);
+    }
 
     // Save the file
     // Only supports java.io.File. TODO: Android Content
-    public boolean saveDocument() {
+    public boolean saveDocument(boolean dontIgnoreEmpty) {
```

Comment:
Suggest passing on the flag named something with "manually" in it. So, it should be possible to save empty file (and possibly future other special cases), only with a manual user button press